---
version: '3.0'

scanning-settings:
  max-depth: 6
  exclude: []
  include-files: []
  system:
    targets: []

security:
  ignore-vulnerabilities:
    62019:
      reason: 'Foca needs to update werkzeug to >=2.3.8.'
      expires: '2025-5-27'
    65213:
      reason: 'Fix is downgrading the pyopenssl package to 21.0.0, doing that breaks all the test.'
      expires: '2025-5-18'
    67599:
      reason: 'No fix available.'
      expires: '2025-5-18'
    70612:
      reason: 'No fix available.'
      expires: '2025-5-18'
    70624:
      reason: 'No fix available.'
      expires: '2025-5-27'
    71595:
      reason: 'Foca needs to update werkzeug to >=2.3.8.'
      expires: '2025-7-21'
    71594:
      reason: 'Foca needs to update werkzeug to >=3.0.3.'
      expires: '2025-7-21'
    71636:
      reason: 'Foca needs to update authlib to >=1.3.0.'
      expires: '2025-7-21'

report:
  dependency-vulnerabilities:
    enabled: true
    auto-ignore-in-report:
      python:
        environment-results: true
        unpinned-requirements: true
      cvss-severity: []

fail-scan-with-exit-code:
  dependency-vulnerabilities:
    enabled: true
    fail-on-any-of:
      cvss-severity:
        - critical
        - high
        - medium
      exploitability:
        - critical
        - high
        - medium

security-updates:
  dependency-vulnerabilities:
    auto-security-updates-limit:
      - patch
...
